dev-packages/e2e-tests/test-applications/nextjs-turbo/.npmrc

```diff
@@ -1,2 +1,4 @@
 @sentry:registry=http://127.0.0.1:4873
 @sentry-internal:registry=http://127.0.0.1:4873
+shamefully-hoist=true
```

Comment:
xD lol

Reply:
m: please add a comment so we know why we added this and that we should reevaluate it from time to time.